### PR TITLE
🔧 Updating dependencies

### PR DESCRIPTION
**Pull request by Sanderand**

### CODE REVIEW

File: src/generator.ts [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+  return {
+    name: node.name.escapedText,
+    isPrivate: false,
+    isTested,
+    branches,
+  };
+};
+
+const sanitizeBranch = (branch: TestBranch): TestBranch => {
+  // note: branches in duplicated conditions will be merged down one level
```
Comment (naflcosta) [inline src/generator.ts]: Should this commented section be left standing here?

File: src/globals/stats.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  filesWritten: 0,
+  testsWritten: 0,
+
+  skipsBecausePrivate: 0,
+  skipsBecauseProtected: 0,
+  skipsBecauseTested: 0,
+
+  started: Date.now(),
+};
```
Comment (naflcosta) [inline src/globals/stats.ts]: Missing empty line at the end?

File: src/generator.ts [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+    fs.writeFileSync(specFilePath, newSpecContent);
+    STATS.filesWritten++;
+  }
+};
+
+export const generateTestsForFile = (filePath: string): void => {
+  STATS.filesRead++;
+
+  const specFilePath: string = filePath.replace('.ts', '.spec.ts');
+  const content = fs.readFileSync(filePath, 'utf-8');
```
Comment (naflcosta) [inline src/generator.ts]: I guess you're using multiple variables declaration convention. Otherwise please correct.

File: src/generator.ts [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+      const nestedBranches: Array<TestBranch> = deepTraverseConditionals(match, indent, conditions);
+      branches.push({ condition: lastCondition, branches: nestedBranches });
+    });
+
+  return branches;
+};
+
+const deepTraverseConditionals = (node: ASTNode, previousIndent: string, previousConditions: Array<string>): Array<TestBranch> => {
+  const indent = previousIndent + SPACER;
+  const conditions: Array<string> = [ ...previousConditions ];
```
Comment (naflcosta) [inline src/generator.ts]: I guess you're using multiple variables declaration convention. Otherwise please correct.

**Comment by Sanderand** [comment]
I've updated the dependencies and adjusted to breaking changes. Also I extracted the code out into different files without changing any of the logic.

**Pull request merged**